063_largestElementInAlist.js

```diff
@@ -0,0 +1,14 @@
+// 63. Write a function that returns the largest element in a list.
```

Comment:
it is a list, not a string. implement the correct logic . The list should be [1, 23, 40, 49, 5] should return 49 as the answer